zoo-hope/src/components/navBar/index.tsx

```diff
@@ -27,17 +33,21 @@ export const NavBar = () => {
             </>
           ) : (
             <Nav.Link href={e.url} className="navbar__link navbar__text">
-              {e.name}
+              {t(e.i18Key)}
             </Nav.Link>
           )
         )}
         <div className="navbar__button-group">
             <button className={`navbar__button-group__UA ${clickedButtonUA ? "navbar__button-group__UA-clicked" : ""}`} onClick={() => {
```

Comment:
`navbar__button-group__UA ${clickedButtonUA ? "navbar__button-group__UA-clicked" : ""}`
Here is better check like this (after changing state)
`navbar__button-group__UA ${langsState === langs.UA ? "navbar__button-group__UA-clicked" : ""}`